### ISSUE TITLE
[Suggestion] Menyoo Customs RGB color entry for wheels

### ISSUE THREAD

**Issue by Gamemassa** [itsjustcurtis/MenyooSP#85]
Hi, I know that in Menyoo Customs you can manually enter RGB/Hex values for the primary and secondary paint colors but there's no option to do that for the wheel colors. Would it be possible to add that in a future update?

**Comment by itsjustcurtis** [comment]
Unfortunately not, wheels do not support RGB colour values, only the pre-defined paint indexes.

**Comment by Gamemassa** [comment]
Hmm, weird. I managed to do it externally with Cheat Engine, so I thought maybe there'd be a way to do it in Menyoo (or some other tool), but I'm not good enough at scripting to know how to add it.

Edit: Not sure if this helps, but what I found in cheat engine was that the wheel color is stored as a 4-byte hexadecimal value that starts with FF (so for example, for Black, it's FF080808). Not sure if there's a way to take RGB values entered into Menyoo and convert them to Hex and then inject _that_, but that could be a workaround.
